Makefile

```diff
@@ -8,7 +8,7 @@ FULL_PIPE_IMAGE_TAG=$(PIPE_IMAGE):$(BRANCH)
 FULL_UI_IMAGE_TAG=$(UI_IMAGE):$(BRANCH)
 SPOKES_FILE ?= ${PWD}/hack/deploy-hub-local/spokes.yaml
 PULL_SECRET ?= ${HOME}/openshift_pull.json
-OCP_VERSION ?= 4.10.9
+OCP_VERSION ?= 4.10.13
```

Comment:
Have in mind that you need to resync the clusterimageset in the hub prior to execute the spoke